roles/rpmdb_verify/tasks/main.yml

```diff
@@ -8,4 +8,4 @@
 - name: Fail if the RPM database is broken
   fail:
     msg: "The RPM database is not usuable"
-  when: rpmdb.stdout <= 0 or rpmdb.rc != 0
+  when: "{{ rpmdb.stdout | int }} <= 0 or rpmdb.rc != 0"
```

Comment:
The output should be a number; I'm using `rpm -qa | wc -l` as the command, so assuming it returns successfully, I believe the integer comparison should work.